utilities/trajectory_generator.py

```diff
@@ -36,13 +37,24 @@ def smooth_path(
     waypoints: List[Pose2d], look_around: float, dist: float, sample_count=10
 ) -> Pose2d:
     """Samples dist meters along waypoints in a way that creates smooth paths"""
-    x, y, angle = 0, 0, 0
+    x, y = 0, 0
+    # takes circular mean https://en.wikipedia.org/wiki/Circular_mean
+    angle_x, angle_y = 0, 0
     for sample in range(sample_count):
         sample_d = dist + scale_value(
             sample, 0, sample_count, -look_around, look_around
         )
         sample_pose = lookup_linear(waypoints, sample_d)
         x += sample_pose.X()
         y += sample_pose.Y()
-        angle += sample_pose.rotation().radians()
-    return Pose2d(x / sample_count, y / sample_count, angle / sample_count)
+        angle_x += sample_pose.rotation().cos()
+        angle_y += sample_pose.rotation().sin()
+    return Pose2d(x / sample_count, y / sample_count, math.atan2(angle_y, angle_x))
+
+
+def ours_to_wpi(pose: Pose2d):
```

Comment:
This is a confusing name. It'd be better if it actually described what it actually does, e.g. "robot pose to field"